rpcs3/rpcs3qt/skylander_dialog.h

```diff
@@ -20,38 +33,16 @@ class skylander_dialog : public QDialog
 	void operator=(skylander_dialog const&) = delete;
 
 protected:
-	// Checks if the skylander is initialized
-	bool is_initialized();
-	// Update the edits from skylander loaded in memory
-	void update_edits();
-	// Parse edits and apply them to skylander in memory
-	void process_edits();
-
-	// Creates a new skylander
-	void new_skylander();
-	// Loads a skylander
-	void load_skylander();
+	void clear_skylander(u8 slot);
+	void create_skylander(u8 slot);
+	void load_skylander(u8 slot);
+	void load_skylander_path(u8 slot, QString path);
```

Comment:
I think this can be const ref. At least I didn't see a reason not to.
```suggestion
	void load_skylander_path(u8 slot, const QString& path);
```